fs/xfstests.py.data/xfstests_64k.yaml

```diff
@@ -0,0 +1,41 @@
+scratch_mnt: '/mnt/scratch'
+test_mnt: '/mnt/test'
+disk_mnt: '/mnt/loop-device'
+
+loop_type: !mux
+    type: 'loop'
+    loop_size: '5GiB'
+    # Option to provide disk for loop device creation,
+    # Uses '/' by default for file creation
+    disk: "null"
+fs_type: !mux
+    fs_ext4_4k:
+        fs: 'ext4'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-b 4096'
+        mount_opt: '-o block_validity'
+    fs_ext4_64k:
+        fs: 'ext4'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-b 65536'
+        mount_opt: '-o block_validity'
+    fs_xfs_4k:
+        fs: 'xfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -b size=4096'
+        mount_opt: ''
+    fs_xfs_64k:
+        fs: 'xfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -b size=65536'
+        mount_opt: ''
+    fs_btrfs_4k:
+        fs: 'btrfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -s 4096 -n 4096'
+        mount_opt: ''
+    fs_btrfs_64k:
+        fs: 'btrfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -s 65536 -n 65536'
+        mount_opt: ''
```

Comment:
similarly 4k sections should be removed if this yaml is only for 64k configuration.